### PR TITLE
IAC AIP Prod Front door config

### PR DESCRIPTION
**Pull request by lewis150193**
New frontend app
Related info:
App: https://github.com/hmcts/ia-aip-frontend
Public DNS Config: https://github.com/hmcts/azure-public-dns/pull/111

Prod Flux Config: https://github.com/hmcts/cnp-flux-config/pull/2654 (requires production-apps / devops team approval)

### CODE REVIEW

File: environments/prod/prod.tfvars [hunk 1]
```diff
@@ -506,5 +506,12 @@ frontends = [
     custom_domain    = "www.apply-for-probate.service.gov.uk"
     backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
     certificate_name = "apply-for-probate-service-gov-uk"
+  },
+  {
+    name             = "ia-aip"
+    mode             = "Detection"
+    custom_domain    = "immigration-appeal.platform.hmcts.net"
```
Comment (timja) [inline environments/prod/prod.tfvars:513]: it seems that in your public DNS PR you've set it up with www, but here you haven't, which is correct?

https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/platform-hmcts-net.yml#L605-L608
https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/platform-hmcts-net.yml#L618-L620

Comment (raghera) [reply environments/prod/prod.tfvars:513]: Hmm - should be without the www.  Shall I change the DNS one?

Comment (raghera) [reply environments/prod/prod.tfvars:513]: will is cause a problem since the DNS one is merged

Comment (raghera) [reply environments/prod/prod.tfvars:728]: changed the dns config

**Pull request merged**